.github/dependabot.yml

```diff
@@ -19,3 +19,21 @@ updates:
     ignore:
       - dependency-name: "au.com.dius.pact*"
         update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
+  - package-ecosystem: "gradle"
```

Comment:
Please add a comment or ensure there is a beefed up description on the PR and resulting commit msg explaining the "why" of this change so we can understand if/when we want to remove this ignore in the future.

Reply:
Updated PR description.